lib/mix/tasks/ash_phoenix.install.ex

```diff
@@ -0,0 +1,42 @@
+if Code.ensure_loaded?(Igniter) do
+  defmodule Mix.Tasks.AshPhoenix.Install do
+    @shortdoc "Installs AshPhoenix into a project. Should be called with `mix igniter.install ash_phoenix`"
+
+    @moduledoc """
+    #{@shortdoc}
+    """
+
+    use Igniter.Mix.Task
+
+    @impl Igniter.Mix.Task
+    def info(_argv, _source) do
+      %Igniter.Mix.Task.Info{}
```

Comment:
Let's set `group: :ash` here, more in preparation for if any options are accepted later.